build.gradle

```diff
@@ -140,15 +140,19 @@ repositories {
     }
 
     maven {
-        url = uri('https://jfrog.sandbox.fynarfin.io/artifactory/fyn-libs-snapshot')
```

Comment:
Changes in this file has to be done because jfrog is not working rn

Reply:
okay, remove this part once jfrog issue is fixed.

Reply:
Okay.